### DISCUSSION TITLE
Black Images / Flash flickering

### DISCUSSION THREAD

**Discussion by jmhwr** [jomjol/AI-on-the-edge-device#1095] [General]
All well so far, but only ~1 out of 10 Images is readable i.e. not black.

The Flash seems to flicker when taking a picture so I guess sometimes the pic is taken when the flash is dark.
I tried the GPIO 4 Settings and disabled the "use interupt" setting but with no difference...

Any ideas?

**Comment by friedpa** [comment]
Do you have enough "Power"? The ESP32 + Cam needs, as far as I rember, at least a 1A power supply. If you have a smaller power supply, you can use a Elko in parallel to stabilies the supply.

**Reply by jmhwr** [reply]
I used a 2.5A supply. For troubleshooting I changed it to a 3A. Let's see what happens...

edit: No, the first pic is pitch black. The flash flashed 3 short times... The next 3 images are also black...

**Comment by ghost** [comment]
Do you set the WaitBeforeTakingPicture to 0?
<img width="877" alt="image" src="https://user-images.githubusercontent.com/412645/192562752-bfac1ee5-adb6-4f3d-b732-fba7123cd6f0.png">

Like @friedpa describes a Elko parallel to the led could be help.

**Reply by jmhwr** [reply]
No, it's set to 5 as in your screenshot. 

To me it seems that the LED is switched on and off in a pattern. When I found the interrupt setting I thought it would be that, because it reminds me of the blinking of a Tx/Rx light or an Act Led on a router...

**Comment by jmhwr** [comment]
Ok guys, I finally tried a new device and it works just fine now.
So it was a hardware default... :-/
